apisix/admin/init.lua

```diff
@@ -130,13 +130,18 @@ local function run()
     local uri_segs = core.utils.split_uri(ngx.var.uri)
     core.log.info("uri: ", core.json.delay_encode(uri_segs))
 
+    local check_only = false
     -- /apisix/admin/schema/route
     local seg_res, seg_id = uri_segs[4], uri_segs[5]
     local seg_sub_path = core.table.concat(uri_segs, "/", 6)
     if seg_res == "schema" and seg_id == "plugins" then
         -- /apisix/admin/schema/plugins/limit-count
         seg_res, seg_id = uri_segs[5], uri_segs[6]
         seg_sub_path = core.table.concat(uri_segs, "/", 7)
+
+    elseif seg_res == "check_conf" then
```

Comment:
I think some comments are needed here to indicate what the check conf interface uri of the Admin API is.

like `-- /apisix/admin/schema/route`